awx/ui/src/screens/Inventory/shared/ConstructedInventoryHint.js

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import { t } from '@lingui/macro';
+import {
+  Alert,
+  AlertActionLink,
+  CodeBlock,
+  CodeBlockAction,
+  CodeBlockCode,
+  ClipboardCopyButton,
+} from '@patternfly/react-core';
+import {
+  TableComposable,
+  Thead,
+  Tr,
+  Th,
+  Tbody,
+  Td,
+} from '@patternfly/react-table';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+
+function ConstructedInventoryHint() {
+  const [copied, setCopied] = React.useState(false);
+
+  const clipboardCopyFunc = (event, text) => {
+    navigator.clipboard.writeText(text.toString());
+  };
+
+  const onClick = (event, text) => {
+    clipboardCopyFunc(event, text);
+    setCopied(true);
+  };
+
+  const pluginSample = `plugin: constructed
+strict: true
+use_vars_plugins: true
+groups:
+  shutdown: resolved_state == "shutdown"
+  shutdown_in_product_dev: resolved_state == "shutdown" and account_alias == "product_dev"
+compose:
+  resolved_state: state | default("running")`;
+
+  return (
+    <Alert
+      isExpandable
+      isInline
+      variant="info"
+      title={t`How to use constructed inventory plugin`}
+      actionLinks={
+        <AlertActionLink
+          href="https://docs.ansible.com/ansible/latest/collections/ansible/builtin/constructed_inventory.html"
+          component="a"
+          target="_blank"
+        >
+          {t`View constructed plugin documentation here`}{' '}
+          <ExternalLinkAltIcon />
+        </AlertActionLink>
+      }
+    >
+      <span>{t`WIP - More to come...`}</span>
```

Comment:
Whats going to be here?

Reply:
I left "wip" there because we haven't nailed down the content. I don't want the hint content to block this PR because the edit form depends on the form to merge. I'll create a follow-up issues just for the ConstructedInventoryHint content. 

@AlanCoding had some ideas of what examples he would like to see in the hint: 
* filtering on hostvars
* filtering on related objects, like inventory name or organization name (via special variables)
* filtering on facts
* some non-trivial combination of logic